### DISCUSSION TITLE
Does anyone know of an extension that highlights when creating functions, or if for loops etc?

### DISCUSSION THREAD

**Discussion by joaocarlosep** [microsoft/vscode-discussions#585] [Extension Show and Tell]
For example, something like a colored background that I tried to express in the image.

![2023-04-26_14h40_22](https://user-images.githubusercontent.com/50813235/234661269-f60c97ed-a39a-4dd6-9ddd-5a9f1cbb8f57.png)

**Comment by alefragnani** [comment]
Hi @joaocarlosep ,

The closest I know is [Blockman](https://marketplace.visualstudio.com/items?itemName=leodevbro.blockman). I think it worth a try 😁 .

Hope this helps
